actionSHARK/mongo.py

```diff
@@ -307,6 +308,46 @@ def __upsert_workflow(self, obj: dict) -> None:
 
         Workflow.objects(**temp_dict).upsert_one(**temp_dict)
 
+    def update_last_updated(self) -> None:
```

Comment:
Instead of updating the last_updated of the VCSSystem, we should create a CISystem collection in the database. If we use updated_at from the VCSSystem we have no way to store updates from the vcsSHARK.